docs/root/configuration/other_features/dlb.rst

```diff
@@ -0,0 +1,39 @@
+.. _config_connection_balance_dlb:
+
+Dlb Connection Balancer
```

Comment:
There should be a release note for this.

Reply:
Added.